a8c-files.php

```diff
@@ -902,23 +905,30 @@ function a8c_files_maybe_inject_image_sizes( $data, $attachment_id ) {
 }
 
 if ( defined( 'FILES_CLIENT_SITE_ID' ) && defined( 'FILES_ACCESS_TOKEN' ) ) {
-	add_action( 'init', 'a8c_files_init' );
-	add_filter( 'intermediate_image_sizes', 'wpcom_intermediate_sizes' );
-	add_filter( 'intermediate_image_sizes_advanced', 'wpcom_intermediate_sizes' );
-	add_filter( 'fallback_intermediate_image_sizes', 'wpcom_intermediate_sizes' );
+	// Conditionally load either the new Stream Wrapper implementation or old school a8c-files
+	if ( defined( 'VIP_FILESYSTEM_USE_STREAM_WRAPPER' ) ) {
```

Comment:
We need to move this inside `a8c_files_init` instead. The code for `intermediate_image_sizes` and `srcset` below are relevant in both situations.

Reply:
Should be fixed with 3537180 but needs more local and production testing.